src/createReducer.ts

```diff
@@ -33,17 +34,19 @@ export default <T extends object>(defaultState: T, rootPath?: string): Reducer<T
 
     return (state: object = defaultState, action: Action): any => {
         if (state && action.type.substr(0, ACTION_PREFIX.length) === ACTION_PREFIX) {
-
             const path = formatPath(action.meta ? action.meta.path : '');
             const parts = path.split('.');
             const newState = { ...state };
-
             // make sure objects don't get mutated by cloning them
             if (parts.length > 1) {
                 cloneObjects(newState, parts, 0);
             }
 
-            return setValue(newState, path, action.payload);
+            if (action.type.substr(0, 6) === '_RESET') {
```

Comment:
Volgens mij klopt dit niet? action.type begint altijd met de ACTION_PREFIX. Er is een functie `actionName()` in createAction.ts, waarmee je de type van de reset action kunt krijgen om te vergelijken.